src/interp/interp.h

```diff
@@ -503,8 +502,10 @@ class Store {
  private:
   template <typename T>
   friend class RefPtr;
+  friend class Thread;
 
   Features features_;
+  Thread *threads_ = nullptr;
```

Comment:
So this is linked list going of threads from the most recently constructed back the oldest?  Maybe add a comment here?

Reply:
Comment added.